src/sql/workbench/contrib/profiler/browser/profilerActions.contribution.ts

```diff
@@ -93,9 +94,23 @@ KeybindingsRegistry.registerCommandAndKeybindingRule({
 			} else {
 				// clear data when profiler is started
 				profilerInput.data.clear();
-				return profilerService.startSession(profilerInput.id, profilerInput.sessionName);
+				return profilerService.startSession(profilerInput.id, profilerInput.sessionName, false);
 			}
 		}
 		return Promise.resolve(false);
 	}
 });
+
+CommandsRegistry.registerCommand({
+	id: 'profiler.openXELFile',
+	handler: (accessor: ServicesAccessor, ...args: any[]) => {
+		let editorService: IEditorService = accessor.get(IEditorService);
+		let fileDialogService: IFileDialogService = accessor.get(IFileDialogService);
+		let profilerService: IProfilerService = accessor.get(IProfilerService);
+		let instantiationService: IInstantiationService = accessor.get(IInstantiationService)
+
+		profilerService.openXELFile(fileDialogService, editorService, instantiationService);
+
+		return Promise.resolve(true);
```

Comment:
Shouldn't we be returning a value from the call to openXELFile instead? Otherwise why bother returning anything at all?